proto/ignition/msgs/bounding_box.proto

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+option java_outer_classname = "BoundingBoxProtos";
+
+/// \ingroup ignition.msgs
+/// \interface BoundingBox
+/// \brief BoundingBox message for object detection annotation
+
+import "ignition/msgs/header.proto";
+
+message BoundingBox
+{
+  /// \brief Optional header data
+  Header header = 1;
+
+  /// \brief Min x coord of the box (top left corner)
+  uint32 minX = 2;
```

Comment:
Is this using integers because the unit is pixels? I think it may be worth it using `double`s because often algorithms working on images return floating point values. Not sure what other people think.

Reply:
+1 for using `double` instead of `uint32` so that this message type can be used across various contexts (it's important to remember that while this PR may be creating a bounding box message type for https://github.com/ignitionrobotics/ign-sensors/issues/135, there may be others who use `ign-msgs` for projects of a different context).